src/cpl/mct/lnd_comp_mct.F90

```diff
@@ -115,6 +117,9 @@ subroutine lnd_init_mct( EClock, cdata_l, x2l_l, l2x_l, NLFilename )
     call seq_cdata_setptrs(cdata_l, ID=LNDID, mpicom=mpicom_lnd, &
          gsMap=GSMap_lnd, dom=dom_l, infodata=infodata)
 
+    call seq_comm_setptrs(LNDID, nthreads=nthreads)
+!$  call omp_set_num_threads(nthreads)
```

Comment:
@jedwards4b you are using what I'd say is a a more obscure syntax for the OMP operations here. I think I'd prefer going with 

#ifdef _OPENMP

which I'd say is more explicit. I'd say change it throughout this file. @billsacks do you have an opinion on this? Without the more explicit #ifdef I think it needs a comment to explain what it is.

Reply:
I don't have strong feelings on it. I had never before seen `!$` used outside of an actual openMP directive, but from a bit of googling it looks like it's a standard way of doing conditional compilation with OpenMP, and I see how it makes sense given that the same form is used in OpenMP directives. It keeps the code cleaner at the cost of being more obscure.

Thinking about it more, though, I guess I'd agree with you @ekluzek that it's probably best to stick with an `#ifdef` here.

For this and your other comment, I'd propose that you or I just do the necessary small changes so @jedwards4b can move on to other things.